scripts/generate-plugins/main.py

```diff
@@ -46,7 +49,7 @@
   'windower': 'Analytics',
   'batchjoiner': 'Analytics',
   'condition': 'Condition',
-  'errortransform': 'Error Handler'
+  'errortransform': 'Error Handler',
```

Comment:
is comma at the end supported in python?

Reply:
done